CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Changed
+- Update version from 0.x.x to 7.0.0 to prep for the GA release
+- Update help screens according to [these guidelines](https://ljfz3b.axshare.com/#id=yokln4&p=conjur_main_help&g=1).
+  [#92](https://github.com/cyberark/conjur-api-python3/issues/92)
```

Comment:
doesn't this need to be `cyberark/conjur-api-python3#92`?

Reply:
Yes updating whole changelog file